packages/sdk/src/health_indicator/health_indicator.spec.ts

```diff
@@ -44,14 +41,13 @@ describe("HealthIndicator", () => {
 
     const changedStatus = await statusChangePromise;
     expect(changedStatus).to.equal(HealthStatus.Unhealthy);
-    expect(healthIndicator.toString()).to.equal(HealthStatus.Unhealthy);
+    expect(healthIndicator.toValue()).to.equal(HealthStatus.Unhealthy);
   });
 
   it("should transition to MinimallyHealthy with one compatible peer", async () => {
     const statusChangePromise = new Promise<HealthStatus>((resolve) => {
-      healthIndicator.addEventListener(
-        HealthStatusChangeEvents.StatusChange,
-        (e: CustomEvent<HealthStatus>) => resolve(e.detail)
+      events.addEventListener("waku:health", (e: CustomEvent<HealthStatus>) =>
```

Comment:
`"waku:health"`

should this be a constant?

Reply:
I thought that it would be better to make `enum` like we did before
but after experimenting with TypeScript and seeing how libp2p does - it's better to make it this way

for example - how many times do you export libp2p event `peer:connect`?